Public/Src/Tools/Xldb.Proto/EnumsAndHelperStructs.proto

```diff
@@ -45,15 +44,17 @@ enum ExecutionEventId{
 }
 
 enum PipOutputOrigin{
-    Produced = 0;
-    UpToDate = 1;
-    DeployedFromCache = 2;
-    NotMaterialized = 3;
+    PipOutputOrigin_Invalid = 0;

```

Comment:
>_Invalid [](start = 19, length = 8)

nit: I don't like '_invalid' suffix. Maybe call it 'uninitialized'...? Don't have a nice suggestion here

Reply:
I like uninitialized, or even _defaultPlaceholder? or _defaultZero?

---
In reply to: [320542914](https://github.com/microsoft/BuildXL/pull/820#discussion_r320542914) [](ancestors = 320542914)